src/openApi/apiResponseModel.ts

```diff
@@ -10,7 +10,12 @@ export class ApiResponse<T = any> {
   constructor(statusCode: number, body?: T, headers?: Record<string, string>) {
     this.body = body;
     this.statusCode = statusCode;
-    this.headers = { 'Content-Type': 'application/hal+json', ...headers };
+    this.headers = {
+      'Access-Control-Expose-Headers':
+        'Location, orion-correlation-id-root, access-control-allow-origin',
```

Comment:
I'm not sure it makes sense to list `access-control-allow-origin` since it is already a CORS header on its own. Unless you have a good reason, I don't think it should be there.

Reply:
I don't think it's a good reason but If I don't include it here I'd need to either include on every get response or otherwise I don't be able to check it in the tests

Reply:
But we are also not setting the `access-control-allow-origin` header in any of the API responses. So making it exposed won't make any difference.

That is actually returned by API GW during the [pre-flight OPTIONS call](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#preflighted_requests).

So if you want to e2e test for that header, it will be about making OPTIONS call and I bet it will be accessible there.

Reply:
Hmm turns out I'm wrong, the underlying opeanpi library sets it https://github.com/Rhosys/openapi-factory.js/blob/release/5.2/src/response.js

So maybe expose it then, it should be case-insensitive but I would anyway match the casing of what openapi-factory.js does to be on the safe side.

Reply:
Added it back